net/nimble/controller/src/ble_ll_adv.c

```diff
@@ -118,13 +119,27 @@ struct ble_ll_adv_sm
 #define BLE_LL_ADV_SM_FLAG_RX_ADD           0x02
 #define BLE_LL_ADV_SM_FLAG_SCAN_REQ_NOTIF   0x04
 #define BLE_LL_ADV_SM_FLAG_CONN_RSP_TXD     0x08
-#define BLE_LL_ADV_SM_FLAG_SECONDARY        0x10
+#define BLE_LL_ADV_SM_FLAG_ACTIVE_CHANSET   0x30 /* use helpers to get/set */
+
+/* active channels set is 1 for primary, 2 for secondary and 0 for none */
+static inline int
+ble_ll_adv_active_chanset_get(struct ble_ll_adv_sm *advsm)
```

Comment:
if you have separate helper for testing primary and secondary you could avoid magic 1 and 2 in code

Reply:
makes sense, will do